src/lib.rs

```diff
@@ -35,12 +35,31 @@ pub struct BitcoinD {
     pub client: Client,
     /// Work directory, where the node store blocks and other stuff. It is kept in the struct so that
```

Comment:
needs update to account for the persisted case

Reply:
Removed the last line all together. More info on `DataDir` is in its own doc and usage info is in `Conf` doc..